clientconfig_test.go

```diff
@@ -20,14 +21,34 @@ nameserver 10.28.10.2
 nameserver 11.28.10.1` // <- NOTE: NO newline.
 
 func testConfig(t *testing.T, data string) {
+	cc, err := ClientConfigFromReader(bytes.NewBufferString(data))
```

Comment:
Why not use `strings.NewReader` here? It's not important but it does avoid an allocation that `bytes.NewBufferString` has to make.

Reply:
Yep, that makes sense.